src/frontend/komponenter/DokumentasjonsbehovListe.tsx

```diff
@@ -35,7 +35,7 @@ export const DokumentasjonsbehovListe: React.FC<IProps> = ({
       <FlexBox>
         <Loader
           size={'xlarge'}
-          title={'Venter på at liste over dokumentasjonsbehov hentes'}
+          title={'Venter på at visning for dokumentasjonsbehov hentes'}
```

Comment:
"visning... hentes" 👀 høres litt rart ut for meg 👀

Reply:
hmmm, kan eventuelt endre til `venter på at dokumentasjonsbehov hentes` da det ikke er en liste som hentes men et enkelt innslag :) 👍

Reply:
[fbc3ca7](https://github.com/navikt/familie-ef-ettersending/pull/518/commits/fbc3ca7cb9e4aa7d050bef617828ed10eec0eccb)